File: model/Identity.vb
Public Class Identity
    Public Seed As Integer
    Public Increment As Integer

    Public Sub New(ByVal seed As Integer, ByVal increment As Integer)
        Me.Seed = seed
        Me.Increment = increment
    End Sub

    Public Function Script() As String
        Return String.Format("IDENTITY ({0},{1})", Seed, Increment)
    End Function

End Class

File: model/Identity.cs
namespace model {
	public class Identity {
		public string Increment;
		public string Seed;

		public Identity(string seed, string increment) {
			Seed = seed;
			Increment = increment;
		}

		public Identity(int seed, int increment) {
			Seed = seed.ToString();
			Increment = increment.ToString();
		}

		public string Script() {
			return string.Format("IDENTITY ({0},{1})", Seed, Increment);
		}
	}
}
